exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let sum = numberOfStudents + numberOfMentors;
+console.log(`Total number of the students and mentors: ${sum}`);
```

Comment:
Very good, you are almost there!

This exercise needs you to output a string like the following:
```
Number of students: 15
Number of mentors: 8
Total numnber of students and mentors: 23
```

You are currently outputting
```
Total number of the students and mentors: 23
```

Also, great usage of backticks! I really like seeing that 👍